Makefile

```diff
@@ -33,7 +33,15 @@ export CGO_ENABLE=0
 # OUT is the directory where dist artifacts and temp files will be created.
 OUT=${GO_TOP}/out
 
-GO ?= go
+# It's more concise to use GO?=$(shell which go)
```

Comment:
I would move this (and other checks/init) into a tools/istio.mk to keep the main makefile more readable ?